ffv1.md

```diff
@@ -516,7 +516,7 @@ Figure: A pseudo-code description of the contexts of Range Non Binary Values. {#
 
 `get_symbol` is used for the read out of `sample_difference` indicated in [@figureSampleDifference].
 
-`get_rac` is the process described in (#range-binary-values).
+`get_rac` returns a sequential boolean value from the bytestream as computed by the process described in (#range-binary-values).
 
```

Comment:
What is a sequential boolean value?

Reply:
I tried to match as much as possible the wording suggested in the review while changing it to the actual type (boolean), but the result is not so good.
new suggestion in the updated commit:
"`get_rac` returns a boolean, computed from the bytestream as described in (#range-binary-values)"
is it fine or do you have a suggestion of better wording?